en/flight_modes/position_mc.md

```diff
@@ -25,18 +25,24 @@ Landing in this mode is easy:
 1. The vehicle will lower propeller thrust, detect the ground and automatically disarm by default (see [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND))
 
 :::warning
-In case the vehicle fails to stop horizontally: Keep it under control in [Altitude](../flight_modes/altitude_mc.md) mode and land like instructed above but manually making sure the vehicle stays above the landing spot using roll and pitch stick. After landing check GPS and magnetometer orientation, calibration.
-
-In case the vehicle fails to detect the ground and landing: After the vehicle is on the ground switch to [Manual/Stabilized](../flight_modes/manual_stabilized_mc.md) mode keeping the throttle stick low and manually disarm using a gesture or other command. Alternatively you can also use the kill switch when the vehicle is already on the ground.
+While very rare on a well calibrated vehicle, sometimes there may be problems with landing.
+- If the vehicle does not stop moving horizontally:
+  - You can still land under control in [Altitude mode](../flight_modes/altitude_mc.md).
+    The approach is the same as above, except that you manually ensure that the vehicle stays above the landing spot using the roll and pitch stick.
+  - After landing check GPS and magnetometer orientation, calibration.
+- If the vehicle does not detect the ground/landing and disarm:
+  - After the vehicle is on the ground switch to [Manual/Stabilized mode](../flight_modes/manual_stabilized_mc.md) keeping the throttle stick low, and manually disarm using a gesture or other command.
+    Alternatively you can also use the kill switch when the vehicle is already on the ground.
 :::
 
 ## Technical Summary
 
-RC/manual mode where RPT sticks control *speed* in corresponding directions. Centered sticks level vehicle and hold it to fixed position and altitude against wind.
+RC mode where roll, pitch, throttle (RPT) sticks control movement in corresponding axes/directions.
+Centered sticks level vehicle and hold it to fixed altitude and position against wind.

```

Comment:
It's probably just technical wise guy nit-pick but nothing guarantees you the vehicle will stay level. It will correct velocity and position and in the desired steady-state case where it arrived very close to the goal position and there's no external disturbance like strong wind it's nearly level but still does some corrections. I think you can judge better how users will intuitively understnad it.

Reply:
@MaEtUgR Hmmm. The point we're trying to make is that if the sticks are centered the vehicle _attempts_ to hold position and attitude. I don't think we're saying that it will always be perfectly level/not do corrections. 

That said, if you think it more accurate we could change:

> Centered sticks level vehicle and hold it to fixed altitude and position against wind.

To

> Centered sticks level the vehicle and actively brake, then and attempt to hold it it's altitude, position and attitude (compensating for wind or other external forces).

Reply:
Thanks for the attempt. That's probably unnecessarily complicated and still has "level the vehicle" so let's just leave it as is. Understandable > wise guy correctness. 😇